packages/codemirror-extension/src/index.ts

```diff
@@ -279,8 +326,10 @@ function activateEditorCommands(
    */
   function isEnabled(): boolean {
     return (
-      tracker.currentWidget !== null &&
-      tracker.currentWidget === app.shell.currentWidget
+      (fileEditorTracker.currentWidget !== null &&
+        fileEditorTracker.currentWidget === app.shell.currentWidget) ||
+      (notebookTracker.currentWidget !== null &&
+        notebookTracker.currentWidget === app.shell.currentWidget)
```

Comment:
Should these be removed? They are in the initial commit that I rebased on top of and I don't fully understand this part. I think that this will mean that some of the fileeditor specific commands below will be erroneously enabled by a notebook being present. (To be clear though I'm not sure)